docs/user-guide/ze-profiles.md

```diff
@@ -67,3 +86,27 @@ Activate the SCS plug-in in Zowe Explorer.
 6. Restart VSCode.
 
 The credentials of your newly created or existing profiles are now stored securely.
+
+## Use Base Profile and Token with Existing Profiles
+
+As a Zowe user, you can leverage the base profile functionality to access multiple Zowe services instantly. Base profiles enable you to authenticate via Zowe API Mediation Layer. You can use base profiles with more than one service profile. Any service profile with the same host and port as your base profile automatically uses your API ML token to log in.
+
+Before using the base profile functionality, ensure that you have [Zowe CLI](https://docs.zowe.org/stable/user-guide/cli-installcli.html) v6.16 or higher installed.
+
+Connect your service profile with a base profile and token.
+
+**Follow these steps:**
+
+1. Open Zowe CLI and run the following command: `zowe auth login apiml`.
+
+2. Follow the instructions to complete the login.
+
+   A local base profile is created that contains your token. For more information about the process, see [Token Management](https://docs.zowe.org/stable/user-guide/cli-usingcli.html#how-token-management-works).
+  
+3. Run Zowe Explorer and click the **+** icon.  
+
+4. Select the profile you use with your base profile with the token.
```

Comment:
I'm not familiar with the functionality in Explorer, but is there a point in this procedure where the user needs to remove host/port from that service profile, or else their command won't get routed through API ML? Should we remind them to do that here? 

Maybe this? "Select the profile that you want to use to access a particular mainframe service. Ensure that the profile does not contain host or port. When you omit host and port, Explorer uses the token in your base profile to connect to API ML and route the command to an appropriate instance of the service. 

Is there a `--base-path` concept in Explorer? Should it be mentioned?

Reply:
@katelynienaber Would you be able to answer the first 2 Brandon's questions? Also, the question about the --base-path concept.